prem_utils/connectors/replicate.py

```diff
@@ -9,6 +15,7 @@ class ReplicateConnector(BaseConnector):
     def __init__(self, api_key: str, prompt_template: str = None):
         super().__init__(prompt_template=prompt_template)
         self.client = Client(api_token=api_key)
+        self.api_key = api_key
```

Comment:
Why this?

Reply:
it's needed because `_upload_and_transform_data` [needs the key to upload the file](https://github.com/premAI-io/prem-utils/blob/feat/fine-tuning/prem_utils/connectors/replicate.py#L135) (no integration in the SDK)